README.rst

```diff
@@ -137,6 +163,50 @@ Or, assuming you have tox installed (via :code:`sudo pip install tox`)::
 
     tox
```

Comment:
I did all the things... and I get a failure on `tests/integrations/test_config.py::test_dcos_url_env_var FAILED`


```
>       assert stderr.startswith(
            b'URL [http://foobar/mesos/master/state.json] is unreachable')
E       assert False
E        +  where False = <built-in method startswith of bytes object at 0x7fe32b890800>(b'URL [http://foobar/mesos/master/state.json] is unreachable')
E        +    where <built-in method startswith of bytes object at 0x7fe32b890800> = b'Traceback (most recent call last):\n  File "/Users/kensipe/projects/dcos-cli/cli/.tox/py35-integration/lib/python3.5...("Expecting value", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n'.startswith

tests/integrations/test_config.py:200: AssertionError
```